src/main/java/seedu/address/ui/MainWindow.java

```diff
@@ -266,23 +274,28 @@ public FocusCard getFocusCard() {
         return focusListPanel;
     }
 
+
     /**
      * Methods where we need to execute the command to get the Index of the Candidate to refresh the panel
      */
     public CommandResult executeCommandThenRefresh(String commandText) throws CommandException, ParseException {
         CommandResult commandResult = logic.execute(commandText);
-        resultDisplay.setFeedbackToUser(commandResult.getFeedbackToUser());
-        if (!focusListPanelPlaceholder.getChildren().isEmpty()) {
-            if (focusListPanel.getCandidate().looseEqual(logic
-                    .getFilteredCandidateList()
-                    .get(commandResult.getEditIndex()))) {
-                executeCommand(FocusCommand.COMMAND_WORD + " "
-                        + String.valueOf(commandResult.getEditIndex() + 1));
-            }
+
+        if (focusListPanel.getCandidate().looseEqual(logic
```

Comment:
will there be an issue if we edit the candidate in the focus panel? will this loose equal still return true?

Reply:
Eh for `EditCommand` it will not use this method! Only for `EditCommand`, we will retrieve the `index` of the `Candidate` first, execute the edit, then we will go back to the list to retrieve the new `Candidate's` information